git-hooks/hooks/pull-update-deps

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Does this need to be duplicated?

Reply:
Sorry, forgot to remove the old directory. Fixed.